cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/upgrade/MetricsDataMigrator.java

```diff
@@ -299,7 +308,7 @@ private MetricsTable getOrCreateMetricsTable(String tableName, DatasetProperties
 
   // todo : use Enum instead of string comparison for the below methods
   private String getMetricNameBasedOnVersion(String metricName, Version version) {
-    if (version.getVersion().equals("2.6")) {
+    if (version == Version.VERSION_26_OR_OLDER) {
       return metricName;
     } else {
```

Comment:
it is usually a good idea to verify the other enum value, and in the last else - fail with "unsupported version" thing